agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/Configuration.java

```diff
@@ -272,6 +272,10 @@ public static class Statsbeat {
     public long longIntervalSeconds = DAYS.toSeconds(1); // default to daily
   }
 
+  public static class PreAggregatedStandardMetricsConfiguration {
+    public boolean disabled = false; // pre-aggregated metrics are on by default
```

Comment:
we generally go with `enabled` flag, even if it's enabled (`true`) by default
```suggestion
    public boolean enabled = true; // pre-aggregated metrics are on by default
```